library/std/src/sys/mod.rs

```diff
@@ -3,6 +3,7 @@
 /// descriptors.
 mod pal;
 
+pub(crate) mod exit_guard;
```

Comment:
Please put this together with the other module declarations below. And this can be `pub`, `sys` is `pub(crate)` already.